src/Components/ImageCheckboxes.tsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { VerticalSpacing } from './VerticalSpacing'
+import { VisuallyHidden } from './VisuallyHidden'
+
+interface ImageCheckboxesProps {
+  id: string
+  values: Array<{ title: string; image: string }>
+  label: string
+}
+
+// Wraps the checkbox area of component
+const CheckboxGroup = styled.div`
+  legend {
+    font-size: ${(props) => props.theme.fontSizes.heading};
+  }
+
+  fieldset {
+    padding: 0;
+    border: none;
+  }
+`
+
+// Contains all the CheckboxItems
+const CheckboxContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 -0.5rem;
+`
+
+// This is the visual representation of the checked status
+const CheckedDisplay = styled.div`
+  border-radius: 1rem;
+  height: 100%;
+  position: absolute;
+  width: 100%;
+`
+
+// This is the text of the checkbox item
+const Text = styled.div`
+  align-items: center;
+  display: flex;
+  font-size: ${(props) => props.theme.fontSizes.normal};
+  font-weight: bold;
+  justify-content: center;
+  letter-spacing: 0.05em;
+  margin: 0 0.5rem 0.5rem 0.5rem;
+  min-height: 3.4rem;
+  z-index: 1;
+`
+
+// Contains label, input, image
+const CheckboxItem = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  flex: 0 0 50%;
+  padding: 0 0.5rem;
+  margin: 0.5rem 0;
+
+  ${(props) => props.theme.screenSizes.tabletPortraitPlus} {
+    flex: 0 0 25%;
+  }
+
+  ${(props) => props.theme.screenSizes.desktopPlus} {
+    flex: 0 0 20%;
+  }
```

Comment:
These media queries will be expecting the page to be a lot wider than it can be here.